### ISSUE TITLE
Support to build non-k8s format images

### ISSUE THREAD

**Issue by kakaZhou719** [alibaba/sealer#1115]
## Issue Description

At present, some user still use other container orchestration tools, such as docker compose in their production environment.

K8s also has officially migration tools, we can integrate those tools to build a k8s format image from the configuration of docker compose. 

even we can build non-k8s format images  with sealer. 

Type: *feature request*

### Additional context
refer : https://kompose.io/

**Comment by fanux** [comment]
Yes, I think the build module don't need to care about kubernetes. CloudImage care about overlayfs, maybe we can do this in nydus file system.

**Comment by kakaZhou719** [comment]
convert docker compose to cloud image rootfs format: sealer convert -t harbor -f docker-compose.yaml
